apps/condo/utils/clientSchema/search.ts

```diff
@@ -0,0 +1,43 @@
+import {
+    GET_ALL_CLASSIFIERS_QUERY,
+    GET_ALL_ORGANIZATION_EMPLOYEE_QUERY,
+    GET_ALL_PROPERTIES_QUERY,
```

Comment:
It confuses me that we have auto-generated queries with a similar name. 
`Ticket.GET_ALL_OBJS_QUERY` with all fields.